tests/63-live-notify_addfd.c

```diff
@@ -0,0 +1,210 @@
+/**
+ * Seccomp Library test program
+ *
+ * Copyright (c) 2025 Microsoft Corporation <sudpandit@microsoft.com>
+ * Author: Sudipta Pandit <sudpandit@microsoft.com>
+ */
+
+/*
+ * This library is free software; you can redistribute it and/or modify it
+ * under the terms of version 2.1 of the GNU Lesser General Public License as
+ * published by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, see <http://www.gnu.org/licenses>.
+ */
+
+#include <errno.h>
+#include <fcntl.h>
+#include <seccomp.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <sys/wait.h>
+#include <unistd.h>
+
+
+int send_fd(int sock, int fd)
+{
+	struct iovec iov = {.iov_base = "F", .iov_len = 1};
+	char buffer[CMSG_SPACE(sizeof(fd))]; // Do i need to set it to zero?
```

Comment:
To answer your question - technically you don't need to zero it out since you're fully populating it below, but this isn't a performance critical path, so I would continue to zero it out.  Good question.